bin/python-noenv.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-env -i /usr/bin/python "$@"
+env -i PBS_DEFAULT=$PBS_DEFAULT /usr/bin/python "$@"
```

Comment:
So, which option do we want to go with?

Change `python-noenv` to only strip out `LD_*` and run with `python -E`?

Should we rename `python-noenv` as well in that case?